bin/webdriver-manager

```diff
@@ -221,6 +226,7 @@ var existingFiles = fs.readdirSync(argv.out_dir);
 
 for (name in binaries) {
   bin = binaries[name];
+  bin.cdn = bin.argv[bin + '_cdn'];
```

Comment:
Well no, since I'm using the CDN on a closed network I'll host the files on the same server.
However, since I noticed that the binaries come from different CDN's I figured I'd add the ability to change them individually.